framework/levure.livecodescript

```diff
@@ -2503,7 +2511,7 @@ Returns: Name
 function levureStandaloneName
   local tName
 
-  if the environment is "development" then
+  if "development" is not in the environment then
```

Comment:
@montegoulding This check should be looking for `is in`, not `is not in`.

Reply:
Ah, goood catch @trevordevore will push a fix in a sec